python/lsst/dm/csc/base/message_director.py

```diff
@@ -63,6 +63,12 @@ def __init__(self, parent, name, config_filename, log_filename):
         self.ARCHIVE_CTRL_CONSUME_QUEUE = None
         self.TELEMETRY_QUEUE = None
 
+        # if wait_for_ack_timeouts is set to True, then a timer is started 
+        # when a message is sent to the Forwarder.  The Forwarder is exxpected
+        # to respond within that timeout window or the archiver will go into
+        # a fault state
+        self.wait_for_ack_timeouts = False
```

Comment:
This is the condition to check whether we want ack timeout or not. If we want configurability, we should put this variable in the config file.

Reply:
I put it in there as a compile-time option as a stop-gap in case this doesn't work out as well as we hope.  If, after testing, it operates the way we expect, another ticket will be filed to get rid of this.